roles/StackStorm.st2/defaults/main.yml

```diff
@@ -34,3 +34,5 @@ st2_auth_username: testu
 st2_auth_password: testp
 # Save credentials in ~/.st2/config file
 st2_save_credentials: yes
+# ST2 packs to be installed (list)
+st2_packs: [azure, "algosec --python3"]
```

Comment:
Per https://github.com/StackStorm/ansible-st2/pull/230#issuecomment-491920683 we want `st2` core pack to be installed by default

Reply:
I am pulling this out (py3)